scripts/ci/publish.ts

```diff
@@ -1321,15 +1324,19 @@ async function sendSlackMessage({
     `${dryRunStr}<https://www.npmjs.com/package/prisma/v/${version}|prisma@${version}> has just been released. Install via \`npm i -g prisma@${version}\` or \`npx prisma@${version}\`
 What's shipped:
 \`prisma/prisma\`
-<https://github.com/prisma/prisma/commit/${prismaCommit.hash}|${prismaLines[0]
+<https://github.com/prisma/prisma/commit/${prismaCommit.hash}|${
+      prismaLines[0]
     }\t\t\t\t-  ${prismaCommit.hash.slice(0, 7)}>
-${prismaLines.slice(1).join('\n')}${prismaLines.length > 1 ? '\n' : ''
+${prismaLines.slice(1).join('\n')}${
+      prismaLines.length > 1 ? '\n' : ''
     }Authored by ${prismaCommit.author}
 
 \`prisma/prisma-engines\`
-<https://github.com/prisma/prisma-engines/commit/${enginesCommit.hash}|${enginesLines[0]
+<https://github.com/prisma/prisma-engines/commit/${enginesCommit.hash}|${
+      enginesLines[0]
     }\t\t\t\t-  ${enginesCommit.hash.slice(0, 7)}>
-${enginesLines.slice(1).join('\n')}${enginesLines.length > 1 ? '\n' : ''
+${enginesLines.slice(1).join('\n')}${
+      enginesLines.length > 1 ? '\n' : ''
     }Authored by ${enginesCommit.author}`,
```

Comment:
Isn't the previous formatting so much better?

Reply:
I agree, but prettier knows so much better about readable code :trollface: